tests/generative/variable_strategies.py

```diff
@@ -164,6 +165,9 @@ def exists(_type, _frame):
     def count(_type, _frame):
         return st.builds(AggregateByPatient.Count, any_frame())
 
+    def count_distinct(type_, _frame):
+        return aggregation_operation(type_, AggregateByPatient.CountDistinct)
```

Comment:
This is very close to being correct, but not quite! `CountDistinct` always returns an `int`, but it can take series of any type. So we can ignore the type we're passed (which will always be `int`) and instead pick a new type to use as the input.
```suggestion
    def count_distinct(_type, _frame):
        return aggregation_operation(any_type(), AggregateByPatient.CountDistinct)
```

(I'm following the convention here that ignored arguments have a leading underscore.)

Reply:
thanks :)